libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -210,6 +211,32 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
         onClick: this.state.responsiveMode && this.props.helpMenuItems ? this.toggleHelpMenu : undefined,
     });
 
+    private getTrialCountdown = (expiredDate: Date) => {
+        const trialDaysLeft = differenceInCalendarDays(expiredDate, new Date()) + 1; // include expired date.
+        if (trialDaysLeft >= 30 && trialDaysLeft <= 90) {
+            const trialMonthsLeft = Math.ceil(trialDaysLeft / 30);
+            if (trialMonthsLeft === 1) {
+                return <FormattedMessage id="gs.header.countdown.oneMonthLeft" />;
+            }
+            return (
+                <FormattedMessage
+                    id="gs.header.countdown.numberOfMonthsLeft"
+                    values={{ number: trialMonthsLeft }}
+                />
+            );
+        } else if (trialDaysLeft > 1 && trialDaysLeft < 30) {
+            return (
+                <FormattedMessage
+                    id="gs.header.countdown.numberOfDaysLeft"
+                    values={{ number: trialDaysLeft }}
+                />
+            );
+        } else if (trialDaysLeft === 1) {
```

Comment:
We don't need `else if` here, do we? There is already a return statement in the preceeding `if` block.
→ Should we replace `else if` with `if`?